src/snowflake/snowpark/session.py

```diff
@@ -4114,6 +4114,7 @@ def call(
         *args: Any,
         statement_params: Optional[Dict[str, Any]] = None,
         log_on_exception: bool = False,
+        is_return_table: Optional[bool] = None,
```

Comment:
do you think `return_dataframe` is a better kwarg here? From a users perspective, I think it makes more sense.

Reply:
I think it is better, do you think we should align it with _call() ?